.gitignore

```
@@ -23,3 +23,6 @@ __pycache__
 # This is the default output directory in dv/uvm/core_ibex and
 # contains auto-generated files from building and running tests.
 /dv/uvm/core_ibex/out
+
+# This is generated by Questa tool when running DV simulations
+/dv/uvm/core_ibex/modelsim.ini
```

This probably can be made slightly wider to catch a `modelsim.ini` anywhere in the tree.

```suggestion
modelsim.ini
```

---

In case you're wondering about the `ucli.key` rule above, that's probably too specific too (I wrote it and haven't got around to changing it).

---

@rswarbrick what do you mean `ucli.key`?